core/src/autogluon/core/trainer/abstract_trainer.py

```diff
@@ -1134,6 +1134,15 @@ def save(self):
         if self.low_memory:
             self.models = models
 
+    def compile_models(self, **kwargs):
+        models_to_compile = list(self.models.keys())
+        if len(models_to_compile) == 0:
+            raise RuntimeError("No valid persisted model to be compiled. "
+                               "Make sure make a call to `persist_models` before trying to compile the models.")
+        for model_name in models_to_compile:
+            self.models[model_name].compile(compiler_configs=kwargs)
```

Comment:
This will probably crash with bagged models. Need to add logic that compiles for bagged_ensemble_model by calling all children's compile methods.

Reply:
Currently we are supporting stacker_ensemble_model, but not bagged_ensemble_model. I still didn't get the idea how to get base_model in bagged_ensemble_model.

Reply:
refer to latest review for clarification